mlx/traceable_base_class.py

```diff
@@ -141,6 +149,12 @@ def set_content(self, content):
             content (str): Content of the item
         '''
         self.content = content
+        if self._state:
```

Comment:
None is not boolean but i dont think it bothers python right?

Reply:
`None` evaluates to `False` if used in a boolean expression. Do you prefer `if self._state is not None:`?

Reply:
No. Evaluqtion was the point